markdown/assignments/pool_b/nahkampf/index.md

```diff
@@ -3,14 +3,24 @@ type: assignment
 title: "Nahkampf"
 author: "Andre Matutat (FH Bielefeld)"
 points: 2
-weight: 1
+weight: 2
 hidden: true
 ---
 
+## Ziel
+
+In dieser Aufgabe implementieren Sie ein einfaches Nahkampfsystem, damit der Held gegen die Monster kämpfen kann.
+
+## Voraussetzung
+
+Um diese Aufgae lösen zu können, müssen Sie vorher `["Monster"]({{< ref "/assignments/pool_b/monster" >}})`{=markdown} implementiert haben.
 
 ## Nahkampf
 
--   Implementieren Sie eine rudimentäre Form eines Kampf-Systems. Dabei soll es zum Kampf kommen, wenn der Held und ein Monster auf demselben Feld stehen.
--   Bei einem erfolgreichen Angriff wird dem angegriffenen Gegner ein bestimmter Schaden zugefügt (dieser verliert dann Lebenspunkte).
--   Ob ein Angriff erfolgreich ist, kann mit einem Zufallsgenerator bestimmt werden: Beispielsweise könnten 60% aller Angriffe erfolgreich sein.
--   Wird der Held von einem Monster getroffen, soll er ein eine kurze Distanz zurückgeschleudert werden. Beachten Sie dabei, dass der Held nicht durch eine Wand bewegt wird.
+Implementieren Sie eine rudimentäre Form eines Kampf-Systems. Dabei soll es zum Kampf kommen, wenn der Held und ein Monster auf demselben Feld stehen (sich berühren).
+
+Bei einem erfolgreichen Angriff wird dem angegriffenen Gegner ein bestimmter Schaden zugefügt (dieser verliert dann Lebenspunkte).
```

Comment:
Der Schaden hängt i.a. von der verwendeten Waffe und der Stärke des Angreifers ab und von der Verteidigungstärke des Gegners. 

Sollten wir das noch ergänzen oder ist das hier zu früh und kommt später nochmal?

Reply:
Im "echten" Spiel gibt es auch Angriffe, wo man keine Lebenspunkte verliert, aber einem beispielsweise die Waffe aus der Hand geschlagen wird ... Oder geht das zu weit?

Reply:
Fällt in meinen Augen beides ehr in Richtung "Item". Schlagen mit einen Schwert ist die Funktion des Item Schwert.

Reply:
Ja, "schlagen" wäre die Funktion im Item "Schwert". Aber trotzdem wird das ja im Kampf ausgeführt.

Und ich denke, dass der Schaden vom verwendeten Item abhängen sollte und auch von der Abwehrstärke des Gegners, das sollte schon nochmal klargestellt werden.

Reply:
Ich verstehe die Punkte. 
Ich zögere nur etwas damit, weil die Monster vermutlich recht früh implementiert werden und daher die wenigsten sowas wie Items etc. schon haben werden.

Reply:
Hab was hinzugefügt.

Reply:
Ja, ich denke das passt jetzt :)